Hackney.Core.sln

```diff
@@ -0,0 +1,158 @@
+EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Hackney.Core.Logging", "Hackney.Core\Hackney.Core.Logging\Hackney.Core.Logging.csproj", "{42BFF25E-520D-4558-8552-FC9B53190426}"
+EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Hackney.Core.Middleware", "Hackney.Core\Hackney.Core.Middleware\Hackney.Core.Middleware.csproj", "{1AD24495-9686-4256-9B21-7C45098A5455}"
+EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Hackney.Core.Sns", "Hackney.Core\Hackney.Core.Sns\Hackney.Core.Sns.csproj", "{FD51B9B0-2220-47B9-88D6-816C56018B32}"
+EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Hackney.Core.Validation", "Hackney.Core\Hackney.Core.Validation\Hackney.Core.Validation.csproj", "{36FEB559-A030-4C8E-8938-EF5A200D3558}"
+EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Hackney.Core.Tests", "Hackney.Core.Tests", "{71B8BB23-0F6E-4B82-B19A-B49C29B3F04C}"
+EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Hackney.Core.Tests.DynamoDb", "Hackney.Core.Tests\Hackney.Core.Tests.DynamoDb\Hackney.Core.Tests.DynamoDb.csproj", "{76AD4430-1C90-41AA-8E48-95A5F8F35BF1}"
+EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Hackney.Core.Tests.HealthCheck", "Hackney.Core.Tests\Hackney.Core.Tests.HealthCheck\Hackney.Core.Tests.HealthCheck.csproj", "{02A6FA16-4EE7-4BD8-B6D0-2B528BCBCFD9}"
+EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Hackney.Core.Tests.Http", "Hackney.Core.Tests\Hackney.Core.Tests.Http\Hackney.Core.Tests.Http.csproj", "{46D34C2B-D03C-4C4F-B8E7-97D11E54CFDA}"
+EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Hackney.Core.Tests.JWT", "Hackney.Core.Tests\Hackney.Core.Tests.JWT\Hackney.Core.Tests.JWT.csproj", "{5DD0ED44-3B91-4475-9444-4E04B002F1C3}"
+EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Hackney.Core.Tests.Logging", "Hackney.Core.Tests\Hackney.Core.Tests.Logging\Hackney.Core.Tests.Logging.csproj", "{3CCA7096-B7BB-44A8-889E-A75D67B9695A}"
+EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Hackney.Core.Tests.Middleware", "Hackney.Core.Tests\Hackney.Core.Tests.Middleware\Hackney.Core.Tests.Middleware.csproj", "{8E807771-7E91-44AD-8769-22CBFD5F8D42}"
+EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Hackney.Core.Tests.Sns", "Hackney.Core.Tests\Hackney.Core.Tests.Sns\Hackney.Core.Tests.Sns.csproj", "{D1834DBC-D796-4FC7-9CAC-31169CCFCCC4}"
+EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Hackney.Core.Tests.Validation", "Hackney.Core.Tests\Hackney.Core.Tests.Validation\Hackney.Core.Tests.Validation.csproj", "{60C7349D-95A8-487C-BA26-CF5B91B00E0D}"
+EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Hackney.Core.Testing", "Hackney.Core\Hackney.Core.Testing\Hackney.Core.Testing.Shared\Hackney.Core.Testing.Shared.csproj", "{47ABA826-9A96-4D5E-9F5E-0A5A5749A1E8}"
+EndProject
```

Comment:
The Hackney.Core.Testing.Shared project should appear in the solution under the Hackney.Core.Testing subfolder, but a the moment it appears under the root.